salt/grains.j2

```diff
@@ -75,18 +75,14 @@ sbd_disk_device: "{{ sbd.device if common.shared_storage_type == "shared-disk" e
 {% endif %}
 
 
-iscsi_enabled: {{ jsonify(true) if common.shared_storage_type == "iscsi" else jsonify(false) }}

```

Comment:
Why have you removed this `iscsi` elements?
I guess we will still need to use iscsi at some cases, they should coexist in my opinion.